sdk/identity/azure-identity/azure/identity/_credentials/authorization_code.py

```diff
@@ -30,6 +30,9 @@ class AuthorizationCodeCredential(GetTokenMixin):
         the authority for Azure Public Cloud (which is the default). :class:`~azure.identity.AzureAuthorityHosts`
         defines authorities for other clouds.
     :keyword str client_secret: One of the application's client secrets. Required only for web apps and web APIs.
+    :keyword list[str] additionally_allowed_tenants: optional additional tenant ids for which the credential
```

Comment:
Have you double-checked the generated docs for these type hints? I just wonder since we usually would provide the type hint as `:paramtype additionally_allowed_tenants: List[str]`, using `List` from `typing`